src/components/Facets/Facets.css

```diff
@@ -0,0 +1,18 @@
+.ds-plp-facets__center-container {
+  width: 100%;
+  max-width: 1600px;
+  margin: 0 auto;
+  padding: 0 12px;
+}
+
+@media (width > 768px) {
+  .ds-plp-facets__center-container {
+    padding: 0 24px;
+  }
+}
+
+@media (width > 1024px) {
+  .ds-plp-facets__center-container {
+    padding: 0 48px;
```

Comment:
tailwind can be used instead ?

Reply:
> tailwind can be used instead ?

Yes, one thing more to remember for me - use tailwind. Sorry, I made appropriate changes.